db/schema.rb

```diff
@@ -11,7 +11,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160410202917) do
+ActiveRecord::Schema.define(version: 20160413023518) do
+
+  create_table "game_histories", force: true do |t|
+    t.integer  "room_id"
+    t.integer  "player_id"
+    t.integer  "game_id"
+    t.string   "game_session_id"
+    t.integer  "player_count"
+    t.integer  "player_team_score"
+    t.integer  "opponent_team_score"
+    t.boolean  "win"
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
Should we have a player_team_id? maybe we could use that to see if team a or team b tends to win more.